torch_np/_dtypes.py

```diff
@@ -343,21 +343,40 @@ def __setstate__(self, value):
 # ### Defaults and dtype discovery
 
 
-def default_int_type():
-    return dtype(_dtypes_impl.default_int_dtype)
-
-
-def default_float_type():
-    return dtype(_dtypes_impl.default_float_dtype)
-
-
-def default_complex_type():
-    return dtype(_dtypes_impl.default_complex_dtype)
-
-
-def get_default_dtype_for(dtyp):
-    torch_dtype = dtype(dtyp).torch_dtype
-    return _dtypes_impl.get_default_type_for(torch_dtype)
+def set_default_dtype(fp_dtype="numpy", int_dtype="numpy"):
+    """Set the (global) defaults for fp and int dtypes.
+
+    Parameters
+    ----------
+    fp_dtype
+        Allowed values are "numpy", "pytorch" or dtype_like things which
+        can be converted into a tnp.DType instance.
```

Comment:
`tnp` is not a standard name. If it accepts PyTorch dtypes let's just say that. No need to be too general.

Reply:
`tnp` is gone in the update, but it's not only pytorch dtypes: it also needs to accept string representations 'float32',  and other numpy-isms (dtype="f8") etc. These conversions are taken care of in `dtype(fp_dtype)` calls.